content/guidelines/reviewing.en.md

```diff
@@ -134,6 +134,13 @@ When tests fail, you can follow these steps to diagnose and address the issue:
 
 4. If the snapshot is the proper one, you can examine the generated version to check the accuracy of content selection.
 
+**Note that if there is at least 1 environment in which the tests run well, we allow to force the merge.** For example, CI tests may fail because of a 403 error but fetch localy the document without errors. We do this because :
```

Comment:
There are no spaces before colons and semicolons in English 🙂 
```suggestion
**Note that if there is at least 1 environment in which the tests run well, we allow to force the merge.** For example, tests may fail in CI because of a 403 Access Denied error, but succeed when run on a development machine. We do this because:
```